### PR TITLE
added "resources" folder as a resource folder to the build

### LINKED ISSUE #1115: Rule Voice Interpreter: missing setting in Paper UI
Paper UI is showing no settings in configuration > services > voice, the block is here but showing no available settings. We should have a setting "item".
As a consequence, the interpreter is no more working.

Can someone please confirm as my server is now patched for voice stuff (but not this part).

### PR DESCRIPTION
**Pull request by kaikreuzer**
fixes #1115
supersedes https://github.com/openhab/openhab-core/pull/1117

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by wborn** [comment]
I hope this doesn't result in an endless build loop in Eclipse like we saw with the resources that had to be removed in https://github.com/openhab/openhab-core/pull/842.

**Comment by kaikreuzer** [comment]
Good point, @wborn. I didn't see any build loop in the IDE during my tests, but I have nonetheless moved it from Maven to bnd now as it solves the issue as well.

File: bundles/org.openhab.core.model.script/bnd.bnd [hunk 1]
```diff
@@ -64,3 +64,4 @@ Require-Bundle: org.antlr.runtime,\
  org.eclipse.xtext.xbase;visibility:=reexport,\
  org.eclipse.xtext;visibility:=reexport,\
  org.objectweb.asm;resolution:=optional
+ -includeresource: -${resources}
```
Comment (wborn) [inline bundles/org.openhab.core.model.script/bnd.bnd]: This line didn't add ESH-INF to the JAR on a Maven build for me.

What did work was:

```suggestion
-includeresource: resources
```

Comment (kaikreuzer) [reply bundles/org.openhab.core.model.script/bnd.bnd]: Not sure why it worked for me, but reading the bnd docs, you are right that this is the correct syntax.

**Pull request merged**